Cargo.toml

```diff
@@ -46,25 +46,30 @@ release_max_level_trace = []
 
 std = []
 
-# requires the latest stable
-# this will have a tighter MSRV before stabilization
-kv_unstable = ["value-bag"]
-kv_unstable_sval = ["kv_unstable", "value-bag/sval", "sval", "sval_ref"]
-kv_unstable_std = ["std", "kv_unstable", "value-bag/error"]
-kv_unstable_serde = ["kv_unstable_std", "value-bag/serde", "serde"]
+kv = []
+kv_sval = ["kv", "value-bag/sval", "sval", "sval_ref"]
+kv_std = ["std", "kv", "value-bag/error"]
+kv_serde = ["kv_std", "value-bag/serde", "serde"]
+
+# Legacy: use `kv_*` instead
+kv_unstable = ["kv"]
+kv_unstable_sval = ["kv_sval"]
+kv_unstable_std = ["kv_std"]
+kv_unstable_serde = ["kv_serde"]
```

Comment:
To bad there is only `compile_error!`, not `compile_warn!` otherwise it would be nice to add that.

Reply:
Actually on second though... we're going to break a lot of (unstable) code already by removing the `as_serde` macros, do we just rip the bandaid in one go and break everything? Or should we add those macros back if [`kv_unstable`] is used (and mark them as `#[deprecated]`)?

Reply:
I think this is a reasonable thing to do :+1:

Unfortunately you can’t mark them as deprecated but we can also re-export the renamed visitor traits under their old names when the old `_unstable` features are enabled.